Add Terms and Conditions section requried by ICU G1. (#3)
derickr/online-safety-assessments: mastodon-terms-phpc.social.rst

=============================================
Only Safety Act Specific Terms and Conditions
=============================================

*To implement "ICU G1 Terms of service: substance (all services)" we have
added the following section into our "about page"
(https://phpc.social/about).*

Handling Illegal Content
------------------------

The moderation team takes illegal content extremely seriously. When being made

> aware about this through Mastodon's reporting features, or third party app

Reviewer: I think "this" should be rephrased to specific wording:
> When being made aware of the presence of illegal content, through Mastodon's reporting features, or third party app reporting features, or through our contact email address, hello@phpcommunity.org, we will immediately take down the illegal content.

=============================================

> Only Safety Act Specific Terms and Conditions

Reviewer 2: ```suggestion
Online Safety Act Specific Terms and Conditions
```

=============================================

*To implement "ICU G1 Terms of service: substance (all services)" we have
added the following section into our "about page"
(https://phpc.social/about).*

Handling Illegal Content
------------------------

> The moderation team takes illegal content extremely seriously. When being made

Reviewer 2: Just a suggestion since "extremely seriously" sounds wordy, and there's no need for the word "being" in "When being made aware..."

```suggestion
The moderation team treats illegal content with the utmost gravity. When made
```
